sprockets/clients/redis/__init__.py

```diff
@@ -109,7 +109,7 @@ def set(self, key, value, ttl=None):
         LOGGER.debug('Setting Redis key "%s"', key)
         ttl = ttl or self.config.ttl
         connection = self._get_shard_connection(key)
-        connection.set(key, value, ex=ttl)
+        connection.setex(name=key, time=ttl, value=value)
```

Comment:
Change this back to `set`.  Per the [Redis docs](http://redis.io/commands/set)

```
Note: Since the SET command options can replace SETNX, SETEX, PSETEX, it is possible that in future versions of Redis these three commands will be deprecated and finally removed.
```